src/main/java/uk/nhs/tis/trainee/notifications/dto/ProgrammeMembershipEvent.java

```diff
@@ -22,14 +22,24 @@
 package uk.nhs.tis.trainee.notifications.dto;
 
 import java.time.Instant;
+import java.time.LocalDate;
+import java.util.List;
 
 /**
  * A Programme Membership event.
  *
  * @param personId            The person associated with the Programme membership.
+ * @param tisId               The TIS ID of the Programme membership.
+ * @param startDate           The start date of the Programme membership.
+ * @param curricula           The list of curricula for the Programme membership.
  * @param conditionsOfJoining The Programme Membership's Conditions of Joining.
  */
-public record ProgrammeMembershipEvent(String personId, ConditionsOfJoining conditionsOfJoining) {
+public record ProgrammeMembershipEvent(
```

Comment:
Reusing this gets quite messy quite quickly I think.

Reply:
I've split it to multiple events for now (it felt less disruptive), but if we end up having to manage other 'flavours' of programme membership then a POJO would be better I think.